lab_1/main.py

```diff
@@ -11,7 +11,13 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if isinstance(text, str):
+        for words in text:
+            if words != ' ' and not words.isalpha():
```

Comment:
if a line is `'a\nboy'`?

Reply:
I checked it and it would be ['aboy'] so I added check for the '\n'